lib/burials/monitor.rb

```diff
@@ -119,5 +129,27 @@ def track_process_attachment_error(in_progress_form, claim, current_user)
       }
       Rails.logger.error('21P-530EZ process attachment error', context)
     end
+
+    ##
+    # log Sidkiq job exhaustion, complete failure after all retries
+    # @see Lighthouse::SubmitBenefitsIntakeClaim
+    #
+    # @param msg [Hash] sidekiq exhaustion response
+    # @param claim [SavedClaim::Burial]
+    #
+    def track_submission_exhaustion(msg, claim = nil)
+      user_account_uuid = msg['args'].length <= 1 ? nil : msg['args'][1]
+      additional_context = {
+        form_id: claim&.form_id,
+        claim_id: msg['args'].first,
+        confirmation_number: claim&.confirmation_number,
+        message: msg
+      }
+      log_silent_failure(additional_context, user_account_uuid, call_location: caller_locations.first)
+
+      StatsD.increment("#{SUBMISSION_STATS_KEY}.exhausted")
+      Rails.logger.error('Lighthouse::SubmitBenefitsIntakeClaim Burial 21P-530EZ submission to LH exhausted!',
```

Comment:
There's a widget in DD that uses Regex `@message_content:21P-530EZ* ` we just have to remember to update it to check for both sides and not just in front now.

Reply:
I believe this would be a new monitor/widget. The earlier one is for the claims controller, this lets us know specifically when the Burial intake job fails